src/components/CountryPicker/index.js

```diff
@@ -35,11 +35,16 @@ function CountryPicker({value, errorText, onInputChange, forwardedRef}) {
     const [isPickerVisible, setIsPickerVisible] = useState(false);
     const [searchValue, setSearchValue] = useState(lodashGet(allCountries, value, ''));
 
+    const updateSearchValue = useCallback((countries, currentValue) => {
+        setSearchValue(lodashGet(countries, currentValue, ''));
+    }, []);
+
     useEffect(() => {
-        setSearchValue(lodashGet(allCountries, value, ''));
-    }, [value, allCountries]);
+        updateSearchValue(allCountries, value);
+    }, [value, allCountries, updateSearchValue]);
```

Comment:
Why we are adding `updateSearchValue` as a new function? We should just call `setSearchValue`

Reply:
Thought it would be DRYer. Will change it.